Main/src/Main/Coffee_Machine.java

```diff
@@ -1,14 +1,31 @@
 package Main;
+import java.util.Scanner;
 
 public class Coffee_Machine {
-    public static void main(String[] args) {
-        System.out.println("Starting to make a coffee");
```

Comment:
Why you deleted old one create new class for new stuff

Reply:
Class is the same as in firs version, I delete old Logic as it was not necessary now.